src/cmake/module/set_plugin_install_rules.cmake

```diff
@@ -11,31 +11,85 @@
+  set_target_properties( ${target} PROPERTIES RUNTIME_OUTPUT_DIRECTORY_RELEASE        ${CMAKE_BINARY_DIR}/${platformType}Release/bin/plugins)
+  set_target_properties( ${target} PROPERTIES RUNTIME_OUTPUT_DIRECTORY_MINSIZEREL     ${CMAKE_BINARY_DIR}/${platformType}MinSizeRel/bin/plugins)
+  set_target_properties( ${target} PROPERTIES RUNTIME_OUTPUT_DIRECTORY_RELWITHDEBINFO ${CMAKE_BINARY_DIR}/${platformType}RelWithDebInfo/bin/plugins)
+  
+  set_target_properties( ${target} PROPERTIES ARCHIVE_OUTPUT_DIRECTORY_DEBUG          ${CMAKE_BINARY_DIR}/${platformType}Debug/bin/plugins)
+  set_target_properties( ${target} PROPERTIES ARCHIVE_OUTPUT_DIRECTORY_RELEASE        ${CMAKE_BINARY_DIR}/${platformType}Release/bin/plugins)
+  set_target_properties( ${target} PROPERTIES ARCHIVE_OUTPUT_DIRECTORY_MINSIZEREL     ${CMAKE_BINARY_DIR}/${platformType}MinSizeRel/bin/plugins)
+  set_target_properties( ${target} PROPERTIES ARCHIVE_OUTPUT_DIRECTORY_RELWITHDEBINFO ${CMAKE_BINARY_DIR}/${platformType}RelWithDebInfo/bin/plugins)
+  
+  set_target_properties( ${target} PROPERTIES LIBRARY_OUTPUT_DIRECTORY_DEBUG          ${CMAKE_BINARY_DIR}/${platformType}Debug/bin/plugins)
+  set_target_properties( ${target} PROPERTIES LIBRARY_OUTPUT_DIRECTORY_RELEASE        ${CMAKE_BINARY_DIR}/${platformType}Release/bin/plugins)
+  set_target_properties( ${target} PROPERTIES LIBRARY_OUTPUT_DIRECTORY_MINSIZEREL     ${CMAKE_BINARY_DIR}/${platformType}MinSizeRel/bin/plugins)
+  set_target_properties( ${target} PROPERTIES LIBRARY_OUTPUT_DIRECTORY_RELWITHDEBINFO ${CMAKE_BINARY_DIR}/${platformType}RelWithDebInfo/bin/plugins)
 else()
-  set_target_properties(${target} PROPERTIES
-    LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/plugins
-    )
+  set_target_properties( ${target} PROPERTIES RUNTIME_OUTPUT_DIRECTORY                ${CMAKE_BINARY_DIR}/bin/plugins)
+  set_target_properties( ${target} PROPERTIES ARCHIVE_OUTPUT_DIRECTORY                ${CMAKE_BINARY_DIR}/bin/plugins)
+  set_target_properties( ${target} PROPERTIES LIBRARY_OUTPUT_DIRECTORY                ${CMAKE_BINARY_DIR}/bin/plugins)
 endif()
 
-
 install(TARGETS ${target}
   RUNTIME DESTINATION bin/plugins
   LIBRARY DESTINATION bin/plugins
   )
 
 endmacro()
+
+
+
+################################################################################
+##############################   LEGACY   ######################################
+
+macro(set_plugin_install_rules_legacy target)
+################################################################################
+#
+# Usage: set_plugin_install_rules_legacy(target)
+# set rules for the plugin legacy designed by the target
+#
+################################################################################
+get_property(GENERATOR_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+
+if(${GENERATOR_MULTI_CONFIG})
+  set_target_properties( ${target} PROPERTIES RUNTIME_OUTPUT_DIRECTORY_DEBUG          ${CMAKE_BINARY_DIR}/${platformType}Debug/bin/plugins_legacy)
```

Comment:
So, you want to keep the plugins in the binary directory? Does it removes the original "plugins" directory at the package root?

Reply:
No decision has been made yet.
We were supposed to talk about it during Monday's videoconference but we did not have time to tackle this point, unfortunately.
I propose to keep this behavior, until we decide for the v3.1 then we fix it all in a dedicate PR.

Reply:
Ok then, it's not a critic problem, it's just a bit weird if you search plugins in the linux package since there are 3 plugins directories. But indeed, it's not an emergency. Perhaps i can create an issue about it to remember the problem?

Reply:
Ok I create the issue #410